### PR TITLE
Fail script if application default credentials are not set

### PR DESCRIPTION
**Pull request by adrcunha**

### CODE REVIEW

File: ci/prow/boskos/set_permissions.sh [hunk 1]
```diff
@@ -78,7 +83,6 @@ done
 echo "Activating GCS service account"
 curl -s -X GET -H "Authorization: Bearer ${ACCESS_TOKEN}" "https://www.googleapis.com/storage/v1/projects/${PROJECT}/serviceAccount"
 GCS_SERVICE_ACCOUNT="service-${PROJECT_NUMBER}@gs-project-accounts.iam.gserviceaccount.com"
-echo "GCS service account is ${GCS_SERVICE_ACCOUNT}"
```
Comment (adrcunha) [inline ci/prow/boskos/set_permissions.sh]: You're probably wondering why I removed this line: the account is already displayed by the output of `curl` above, so this is just a redundant log.

File: ci/prow/boskos/set_permissions.sh [hunk 2]
```diff
@@ -78,7 +83,6 @@ done
 echo "Activating GCS service account"
 curl -s -X GET -H "Authorization: Bearer ${ACCESS_TOKEN}" "https://www.googleapis.com/storage/v1/projects/${PROJECT}/serviceAccount"
 GCS_SERVICE_ACCOUNT="service-${PROJECT_NUMBER}@gs-project-accounts.iam.gserviceaccount.com"
```
Comment (chizhg) [inline ci/prow/boskos/set_permissions.sh]: Then also remove this line?

Comment (adrcunha) [reply ci/prow/boskos/set_permissions.sh]:
```suggestion
```

Comment (adrcunha) [reply ci/prow/boskos/set_permissions.sh]: I messed up my branch, this line was added back and I missed it. :P 
Removed, thanks!

File: ci/prow/boskos/set_permissions.sh (hunk 1 above)
Comment (chizhg) [reply ci/prow/boskos/set_permissions.sh]: /lgtm

**Pull request merged**